### DISCUSSION TITLE
你推薦的網路文章平台

### DISCUSSION THREAD

**Discussion by forestdew507** [1101-GS4719-C/Python-Community-GS4719-C#2150] [Q&A]
好奇大家有沒有推薦/常用的網路文章寫作社群平台？（創作者友善 or 閱讀者友善都可）

*

上學期修了外系的課程，在克難完成一個不是本科系的個人期末專題的過程，我收穫了網路平台 Medium。對我來說，和大學生更常使用的 IG、FB、脆相比，Medium 是一個作者創作篇幅更長、文章資訊專業含量也更高的 social media。（前幾天才遲鈍意識到這類的社群平台該被歸到「網路文章」平台才對XD 也就是 IG、FB、Threads 歸在一類、Medium、方格子在一類，得算是不同類的相比哈哈。）

不過看到網文提到，相對來說，Medium 對中文文章比英文文章較不友善，從身為閱讀者的視角來看滿有感的，基本上如果不用關鍵字搜尋，主頁顯示的都是英文文章。

*

無論是想身為文章能被更多人看見的中文創作者，或者身為想觸及中文文章的讀者，求除了版面賞心悅目的 Medium 外，還有沒有其他網路文章推薦的平台 ! thanks:)

**Comment by Jinyuu17** [comment]
如果是中文的文章，不妨考慮對岸的一些網頁，畢竟他們佔據了中文人口的一大部分。簡書、知乎、豆瓣、微信公眾號都不錯。

**Comment by zoo100130** [comment]
X(前推特)、Reddit應該算在世界都算鰻熱門的social media，朋友去日本玩被高中生搭訕他們也是都用推特，也不乏有含金量很高的文章類型，個人覺得算是萬金油類型的軟體，日常社交、知識傳遞、政令宣導都很好用。
